e2e/node/e2e.test.ts

```diff
@@ -35,17 +35,18 @@ import { Session } from "@inrupt/solid-client-authn-node";
 import { isVerifiableCredential } from "@inrupt/solid-client-vc";
 import { getNodeTestingEnvironment } from "@inrupt/internal-test-env";
 import {
+  AccessGrant,
```

Comment:
There are some sorting changes here. If we keep the lists sorted, and we don't just add new imports to the end of an import list, it minimizes the chance for a merge conflicts. And it looks nice :)